invisible_cities/reco/corrections_test.py

```diff
@@ -360,9 +361,10 @@ def test_corrections_1d(gauss_data_1d):
     mean = np.mean(Eevt)
     std  = np.std (Eevt)
 
-    y, x = np.histogram(Eevt, np.linspace(mean - 3 * std, mean + 3 * std, 100))
-    x    = x[:-1] + np.diff(x) * 0.5
-    f    = fitf.fit(fitf.gauss, x, y, (1e5, mean, std))
+    y, x  = np.histogram(Eevt, np.linspace(mean - 3 * std, mean + 3 * std, 100))
+    x     = x[:-1] + np.diff(x) * 0.5
```

Comment:
This is the same as the function `shift_to_bin_centers` in the `icaro` package. Can we move it also to `core` and use it here?